bin/docker-entrypoint

```diff
@@ -1,24 +1,8 @@
 #!/bin/bash
 set -e
 
-if [ "$RAILS_ENV" != "production" ]; then
-  echo "In a non-production environment: initializing database"
-  bundle exec rake db:create db:schema:load
-else
-  DB_STATUS=$(bundle exec rake db:migrate:status 2>&1 | tail -n 1)
+echo "Running migrations..."
+bundle exec rails db:setup_or_migrate
 
-  if [ "$DB_STATUS" == "Schema migrations table does not exist yet."  ]; then
-    echo "No tables exist - setting up the database"
-    bundle exec rake db:setup
-  else
-    echo "Running the schema and data migrations"
-    bundle exec rails db:migrate
-  fi
-fi
-
-if [ "$RAILS_ENV" != "test" ]; then
-  echo "Scheduling Cron Jobs"
-  bundle exec rake jobs:schedule
-fi
-
-exec "$@"
+echo "Scheduling jobs..."
```

Comment:
Sorry, missed this on a previous review but you've lost the `if [ "$RAILS_ENV" != "test" ];` here.

Reply:
Yeah, we don't need this as `bin/prepare-database` doesn't get used in test.

Reply:
(See https://github.com/DFE-Digital/dfe-teachers-payment-service/pull/831/files#diff-0d4080c652c491057176df93ca5768d1R23)

Reply:
In the context of the commit where you introduce the `db:setup_or_migrate` task, `bin/docker-entrypoint` is still being used in the test Rails env, so this check needs to remain there in that commit.

Reply:
Done! 👍

Reply:
Oh, I've realised that the same commit ("Make database setup command more robust") also accidentally loses the `exec "$@"`; could you restore that too please?

Reply:
Done! 👍